### PR TITLE
Upscaled Icons

### PR DESCRIPTION
**Pull request by dvaldivia**
Uses now standarized 256x256 icons
Screen Shot 2021-08-31 at 3 02 58 PM

### CODE REVIEW

File: portal-ui/src/screens/Console/Account/Account.tsx [hunk 1]
```diff
@@ -249,7 +249,7 @@ const Account = ({
                 disableUnderline: true,
                 startAdornment: (
                   <InputAdornment position="start">
-                    <SearchIcon width={18} />
+                    <SearchIcon style={{ width: 16, height: 16 }} />
```
Comment (bexsoft) [inline portal-ui/src/screens/Console/Account/Account.tsx:252]: Avoid using inline styling. Please try to use a class instead.

File: portal-ui/src/screens/Console/Buckets/ListBuckets/ListBuckets.tsx [hunk 2]
```diff
@@ -266,7 +266,7 @@ const ListBuckets = ({
                 disableUnderline: true,
                 startAdornment: (
                   <InputAdornment position="start">
-                    <SearchIcon width={18} />
+                    <SearchIcon style={{ width: 16, height: 16 }} />
```
Comment (bexsoft) [inline portal-ui/src/screens/Console/Buckets/ListBuckets/ListBuckets.tsx:269]: Avoid using inline styling. Please try to use a class instead.

File: portal-ui/src/screens/Console/Buckets/ListBuckets/Objects/ListObjects/ListObjects.tsx [hunk 3]
```diff
@@ -923,7 +925,7 @@ const ListObjects = ({
               disableUnderline: true,
               startAdornment: (
                 <InputAdornment position="start">
-                  <SearchIcon width={18} />
+                  <SearchIcon style={{ width: 16, height: 16 }} />
```
Comment (bexsoft) [inline portal-ui/src/screens/Console/Buckets/ListBuckets/Objects/ListObjects/ListObjects.tsx:928]: Avoid using inline styling. Please try to use a class instead.

File: portal-ui/src/screens/Console/Buckets/ListBuckets/Objects/ObjectDetails/ObjectDetails.tsx [hunk 4]
```diff
@@ -761,7 +761,7 @@ const ObjectDetails = ({
                         disableUnderline: true,
                         startAdornment: (
                           <InputAdornment position="start">
-                            <SearchIcon width={18} />
+                            <SearchIcon style={{ width: 16, height: 16 }} />
```
Comment (bexsoft) [inline portal-ui/src/screens/Console/Buckets/ListBuckets/Objects/ObjectDetails/ObjectDetails.tsx:764]: Avoid using inline styling. Please try to use a class instead.

File: portal-ui/src/screens/Console/Configurations/ConfigurationPanels/WebhookPanel.tsx [hunk 5]
```diff
@@ -147,7 +147,7 @@ const WebhookPanel = ({ match, classes }: IWebhookPanel) => {
               disableUnderline: true,
               startAdornment: (
                 <InputAdornment position="start">
-                  <SearchIcon width={18} />
+                  <SearchIcon style={{ width: 16, height: 16 }} />
```
Comment (bexsoft) [inline portal-ui/src/screens/Console/Configurations/ConfigurationPanels/WebhookPanel.tsx:150]: Avoid using inline styling. Please try to use a class instead.

File: portal-ui/src/screens/Console/Configurations/NotificationEndpoints/ListNotificationEndpoints.tsx [hunk 6]
```diff
@@ -183,7 +183,7 @@ const ListNotificationEndpoints = ({
                             disableUnderline: true,
                             startAdornment: (
                               <InputAdornment position="start">
-                                <SearchIcon width={18} />
+                                <SearchIcon style={{ width: 16, height: 16 }} />
```
Comment (bexsoft) [inline portal-ui/src/screens/Console/Configurations/NotificationEndpoints/ListNotificationEndpoints.tsx:186]: Avoid using inline styling. Please try to use a class instead.

File: portal-ui/src/screens/Console/Configurations/TiersConfiguration/ListTiersConfiguration.tsx [hunk 7]
```diff
@@ -228,7 +228,7 @@ const ListTiersConfiguration = ({
                             disableUnderline: true,
                             startAdornment: (
                               <InputAdornment position="start">
-                                <SearchIcon width={18} />
+                                <SearchIcon style={{ width: 16, height: 16 }} />
```
Comment (bexsoft) [inline portal-ui/src/screens/Console/Configurations/TiersConfiguration/ListTiersConfiguration.tsx:231]: Avoid using inline styling. Please try to use a class instead.

File: portal-ui/src/screens/Console/DirectCSI/DirectCSIDrives.tsx [hunk 8]
```diff
@@ -260,7 +260,7 @@ const DirectCSIMain = ({
             disableUnderline: true,
             startAdornment: (
               <InputAdornment position="start">
-                <SearchIcon width={18} />
+                <SearchIcon style={{ width: 16, height: 16 }} />
```
Comment (bexsoft) [inline portal-ui/src/screens/Console/DirectCSI/DirectCSIDrives.tsx:263]: Avoid using inline styling. Please try to use a class instead.

File: portal-ui/src/screens/Console/DirectCSI/DirectCSIVolumes.tsx [hunk 9]
```diff
@@ -117,7 +117,7 @@ const DirectCSIVolumes = ({
             disableUnderline: true,
             startAdornment: (
               <InputAdornment position="start">
-                <SearchIcon width={18} />
+                <SearchIcon style={{ width: 16, height: 16 }} />
```
Comment (bexsoft) [inline portal-ui/src/screens/Console/DirectCSI/DirectCSIVolumes.tsx:120]: Avoid using inline styling. Please try to use a class instead.

File: portal-ui/src/screens/Console/Groups/Groups.tsx [hunk 10]
```diff
@@ -198,7 +198,7 @@ const Groups = ({ classes, setErrorSnackMessage }: IGroupsProps) => {
                 disableUnderline: true,
                 startAdornment: (
                   <InputAdornment position="start">
-                    <SearchIcon width={18} />
+                    <SearchIcon style={{ width: 16, height: 16 }} />
```
Comment (bexsoft) [inline portal-ui/src/screens/Console/Groups/Groups.tsx:201]: Avoid using inline styling. Please try to use a class instead.

File: portal-ui/src/screens/Console/Groups/UsersSelectors.tsx [hunk 11]
```diff
@@ -196,7 +196,7 @@ const UsersSelectors = ({
                   InputProps={{
                     startAdornment: (
                       <InputAdornment position="start">
-                        <SearchIcon width={18} />
+                        <SearchIcon style={{ width: 16, height: 16 }} />
```
Comment (bexsoft) [inline portal-ui/src/screens/Console/Groups/UsersSelectors.tsx:199]: Avoid using inline styling. Please try to use a class instead.

File: portal-ui/src/screens/Console/Tenants/TenantDetails/pods/PodLogs.tsx [hunk 12]
```diff
@@ -182,7 +182,7 @@ const PodLogs = ({
             disableUnderline: true,
             startAdornment: (
               <InputAdornment position="start">
-                <SearchIcon width={18} />
+                <SearchIcon style={{ width: 16, height: 16 }} />
```
Comment (bexsoft) [inline portal-ui/src/screens/Console/Tenants/TenantDetails/pods/PodLogs.tsx:185]: Avoid using inline styling. Please try to use a class instead.

File: portal-ui/src/screens/Console/Users/GroupsSelectors.tsx [hunk 13]
```diff
@@ -189,7 +189,7 @@ const GroupsSelectors = ({
                 InputProps={{
                   startAdornment: (
                     <InputAdornment position="start">
-                      <SearchIcon width={18} />
+                      <SearchIcon style={{ width: 16, height: 16 }} />
```
Comment (bexsoft) [inline portal-ui/src/screens/Console/Users/GroupsSelectors.tsx:192]: Avoid using inline styling. Please try to use a class instead.

File: portal-ui/src/screens/Console/Users/ListUsers.tsx [hunk 14]
```diff
@@ -228,7 +228,7 @@ const ListUsers = ({ classes, setErrorSnackMessage, history }: IUsersProps) => {
                 disableUnderline: true,
                 startAdornment: (
                   <InputAdornment position="start">
-                    <SearchIcon width={18} />
+                    <SearchIcon style={{ width: 16, height: 16 }} />
```
Comment (bexsoft) [inline portal-ui/src/screens/Console/Users/ListUsers.tsx:231]: Avoid using inline styling. Please try to use a class instead.

File: portal-ui/src/screens/Console/Tenants/TenantDetails/PoolsSummary.tsx [hunk 15]
```diff
@@ -119,7 +119,7 @@ const PoolsSummary = ({
               disableUnderline: true,
               startAdornment: (
                 <InputAdornment position="start">
-                  <SearchIcon width={18} />
+                  <SearchIcon style={{ width: 16, height: 16 }} />
```
Comment (bexsoft) [inline portal-ui/src/screens/Console/Tenants/TenantDetails/PoolsSummary.tsx:122]: Avoid using inline styling. Please try to use a class instead.

File: portal-ui/src/screens/Console/Storage/StoragePVCs.tsx [hunk 16]
```diff
@@ -109,7 +109,7 @@ const StorageVolumes = ({
             disableUnderline: true,
             startAdornment: (
               <InputAdornment position="start">
-                <SearchIcon width={18} />
+                <SearchIcon style={{ width: 16, height: 16 }} />
```
Comment (bexsoft) [inline portal-ui/src/screens/Console/Storage/StoragePVCs.tsx:112]: Avoid using inline styling. Please try to use a class instead.

File: portal-ui/src/screens/Console/Policies/PolicySelectors.tsx [hunk 17]
```diff
@@ -184,7 +184,7 @@ const PolicySelectors = ({
                 InputProps={{
                   startAdornment: (
                     <InputAdornment position="start">
-                      <SearchIcon width={18} />
+                      <SearchIcon style={{ width: 16, height: 16 }} />
```
Comment (bexsoft) [inline portal-ui/src/screens/Console/Policies/PolicySelectors.tsx:187]: Avoid using inline styling. Please try to use a class instead.

File: portal-ui/src/screens/Console/Policies/PolicyDetails.tsx [hunk 18]
```diff
@@ -491,7 +491,7 @@ const PolicyDetails = ({
                       disableUnderline: true,
                       startAdornment: (
                         <InputAdornment position="start">
-                          <SearchIcon width={18} />
+                          <SearchIcon style={{ width: 16, height: 16 }} />
```
Comment (bexsoft) [inline portal-ui/src/screens/Console/Policies/PolicyDetails.tsx:494]: Avoid using inline styling. Please try to use a class instead.

File: portal-ui/src/screens/Console/Policies/ListPolicies.tsx [hunk 19]
```diff
@@ -188,7 +188,7 @@ const ListPolicies = ({ classes, setErrorSnackMessage }: IPoliciesProps) => {
                 disableUnderline: true,
                 startAdornment: (
                   <InputAdornment position="start">
-                    <SearchIcon width={18} />
+                    <SearchIcon style={{ width: 16, height: 16 }} />
```
Comment (bexsoft) [inline portal-ui/src/screens/Console/Policies/ListPolicies.tsx:191]: Avoid using inline styling. Please try to use a class instead.

File: portal-ui/src/screens/Console/ObjectBrowser/BrowseBuckets.tsx [hunk 20]
```diff
@@ -275,7 +280,7 @@ const BrowseBuckets = ({
               disableUnderline: true,
               startAdornment: (
                 <InputAdornment position="start">
-                  <SearchIcon width={18} />
+                  <SearchIcon style={{ width: 16, height: 16 }} />
```
Comment (bexsoft) [inline portal-ui/src/screens/Console/ObjectBrowser/BrowseBuckets.tsx:283]: Avoid using inline styling. Please try to use a class instead.

File: portal-ui/src/screens/Console/Logs/ErrorLogs/ErrorLogs.tsx [hunk 21]
```diff
@@ -337,7 +337,7 @@ const ErrorLogs = ({
               disableUnderline: true,
               startAdornment: (
                 <InputAdornment position="start">
-                  <SearchIcon width={18} />
+                  <SearchIcon style={{ width: 16, height: 16 }} />
```
Comment (bexsoft) [inline portal-ui/src/screens/Console/Logs/ErrorLogs/ErrorLogs.tsx:340]: Avoid using inline styling. Please try to use a class instead.

**Review by bexsoft — CHANGES_REQUESTED** [review]

**Review by bexsoft — APPROVED** [review]
LGTM

**Review by Alevsk — APPROVED** [review]

**Pull request merged**